src/stories/mock/index.tsx

```diff
@@ -0,0 +1,5 @@
+import fetchMock from 'fetch-mock';
+export { followers } from './followers';
+export { subscriber } from './subscriber';
+export { users } from './users';
+export default fetchMock;
```

Comment:
Quick simplification
```suggestion
export { default } from 'fetch-mock';
export { followers } from './followers';
export { subscriber } from './subscriber';
export { users } from './users';
```